src/js/timeline.js

```diff
@@ -1,80 +1,49 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Tweet extends React.Component {
+  render() {
+    let date = new Date(this.props.content.createdAt);
+    let formattedDate = date.toLocaleDateString("en-US", {month:"short", day:"2-digit"});
+    let user = this.props.content.twitterUser;
+    let content = this.props.content;
+    return React.createElement('div', {className: 'tweet'},
+      React.createElement('div', {className: 'userInfo'},
+        React.createElement('div', {className: 'img'},
+          React.createElement('img', {src: `${user.profileImageUrl}`, className: 'img-circle'}, null)),
+        React.createElement('div', {className: 'name'}, `${user.name}`),
+        React.createElement('div', {className: 'handle'}, `@${user.twitterHandle}`)
+      ),
+      React.createElement('div', {className: 'content'},
+        React.createElement('div', {className: 'date'}, formattedDate),
+        React.createElement('a', {className: 'link', target: '_blank', href: `https://twitter.com/${user.twitterHandle}/status/${content.id}`},
+          React.createElement('div', {className: 'msg'}, `${content.message}`))
+      )
+    );
+  }
+}
+
+class Tweets extends React.Component {
+  render() {
+    let tweets = [];
+    let timelineList = this.props.timelineList;
+    for(let i = 0; i < timelineList.length; i++) {
+      let content= timelineList[i];
```

Comment:
add space after content

Reply:
it's better to create a 'props' object, fill it in and then pass that in the method
i.e. tweets.push(React.createElement(Tweet, props}). I believe you don't need to explicitly pass in null